app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+    <div>Currently waiting confirmation for: <%= resource.unconfirmed_email %></div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+    <% if @minimum_password_length %>
+      <br />
+      <em><%= @minimum_password_length %> characters minimum</em>
+    <% end %>
+  </div>
+
```

Comment:
* After successfully installing and running your app, i couldn't perform any of these functionalities. Kindly fix this as its a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Can%20register%20a,and%20your%20styling.)

> 

> Can register a new user.
> User logs in with a combination of email and password.
> Hashed password should be stored in the database.
> Ask for confirmation of email.
> Can reset password.

Reply:
Hello, @Davidosky007 😁
All the requirements can be performed in the app on my side as seen in the following [recording](https://www.loom.com/share/bb33a9c0328a41a8b71784fb245a9433).
Do you have any idea of what can be causing the issue on your side? 🤔